the-example-app/README.md

```diff
@@ -24,14 +24,14 @@ And then retry the upper code again.
 
 
 
-Other apps
+Compatible apps
 ==========
 
-The Node example app is not the only one. Soon™ there will be more available, all using the
-same space.
-
-Stay tuned for updates on the model and example apps.
+The content model `The example app` is compatible with platform specific example apps. It works alongside the apps source code and forms a reference Contentful setup.
```

Comment:
"The content model `The example app` is used in platform specific example apps to form a reference Contentful setup."

A little more succinct. WDYT?